src/lib/styles/postcss-configuration.ts

```diff
@@ -90,7 +88,7 @@ export async function loadPostcssConfiguration(
 
   const config: PostcssConfiguration = { plugins: [] };
   for (const [name, options] of entries) {
-    if (!options || typeof options !== 'object') {
+    if (!options || typeof options !== 'object' || typeof raw.plugins !== 'string') {
```

Comment:
I am a bit confused by this condition; should it be something like

```suggestion
    if (!options || (typeof options !== 'object' && typeof options  !== 'string')) {
```

Reply:
You are right